packages/theme-default/src/markdown/highlighter.ts

```diff
@@ -17,12 +17,12 @@ const __dirname = fileURLToPath(new URL('.', import.meta.url))
 const nightOwl = JSON.parse(readFileSync(resolve(__dirname, './night-owl.json'), 'utf-8'))
 const vitesseLight = JSON.parse(readFileSync(resolve(__dirname, './vitesse-light.json'), 'utf-8'))
 
-async function createHighlighterWithThemeAndLangs(theme: any, langs: Lang[]) {
+async function createHighlighterWithThemeAndLangs(theme: any, langs: BundledLanguage[]) {
   const shikiHighlighter = await getHighlighter({
-    theme,
+    themes: ['vitesse-light', 'night-owl'],
     langs,
   })
```

Comment:
Wait, so the "fix" for Shiki v1 support was to hardcode two themes? Using any other Shiki theme (e.g. `github-light` and `github-dark`) now results in this error:

```
7:06:00 AM [vite] Internal server error: Theme `github-light` not found, you may need to load it first
  Plugin: @sveltepress/vite
  File: /Users/gka/projects/svelteplot/src/routes/introduction/+page.md
      at getTheme (file:///Users/gka/projects/svelteplot/node_modules/.pnpm/@shikijs+core@1.1.1/node_modules/@shikijs/core/dist/index.mjs:5292:19)
      at Object.setTheme (file:///Users/gka/projects/svelteplot/node_modules/.pnpm/@shikijs+core@1.1.1/node_modules/@shikijs/core/dist/index.mjs:5296:23)
      at codeToTokensBase (file:///Users/gka/projects/svelteplot/node_modules/.pnpm/@shikijs+core@1.1.1/node_modules/@shikijs/core/dist/index.mjs:516:42)
      at codeToTokens (file:///Users/gka/projects/svelteplot/node_modules/.pnpm/@shikijs+core@1.1.1/node_modules/@shikijs/core/dist/index.mjs:759:18)
      at codeToHast (file:///Users/gka/projects/svelteplot/node_modules/.pnpm/@shikijs+core@1.1.1/node_modules/@shikijs/core/dist/index.mjs:991:53)
      at codeToHtml (file:///Users/gka/projects/svelteplot/node_modules/.pnpm/@shikijs+core@1.1.1/node_modules/@shikijs/core/dist/index.mjs:4552:25)
      at Object.codeToHtml (file:///Users/gka/projects/svelteplot/node_modules/.pnpm/@shikijs+core@1.1.1/node_modules/@shikijs/core/dist/index.mjs:5345:40)
      at Object.highlighter [as light] (file:///Users/gka/projects/svelteplot/node_modules/.pnpm/@sveltepress+theme-default@1.22.4_@algolia+client-search@4.22.1_@sveltejs+kit@2.5.0_@sveltejs_7j6agbtbqcdm3tu7rocwzcwapq/node_modules/@sveltepress/theme-default/dist/markdown/highlighter.js:60:83)
      at file:///Users/gka/projects/svelteplot/node_modules/.pnpm/@sveltepress+theme-default@1.22.4_@algolia+client-search@4.22.1_@sveltejs+kit@2.5.0_@sveltejs_7j6agbtbqcdm3tu7rocwzcwapq/node_modules/@sveltepress/theme-default/dist/markdown/highlighter.js:126:107
      at step (file:///Users/gka/projects/svelteplot/node_modules/.pnpm/@sveltepress+theme-default@1.22.4_@algolia+client-search@4.22.1_@sveltejs+kit@2.5.0_@sveltejs_7j6agbtbqcdm3tu7rocwzcwapq/node_modules/@sveltepress/theme-default/dist/markdown/highlighter.js:32:23)
```

Reply:
I think instead of hardcoding the `['vitesse-light', 'night-owl']` themes, we should use the themes defined in the sveltepress config via `highlighter.themeLight` and `highlighter.themeDark`.